packages/boa/tools/clean-python.js

```diff
@@ -0,0 +1,18 @@
+'use strict';
+
+const { run, CONDA_DOWNLOAD_NAME, CONDA_INSTALL_DIR } = require('./utils');
+const fs = require('fs');
+
+let CONDA_DOWNLOAD_PREFIX = 'https://repo.anaconda.com/miniconda';
+if (process.env.BOA_CONDA_MIRROR) {
+  CONDA_DOWNLOAD_PREFIX = process.env.BOA_CONDA_MIRROR;
+}
+
+if (fs.existsSync(CONDA_DOWNLOAD_NAME)) {
+  run('rm', `-f ./${CONDA_DOWNLOAD_NAME}`);
+}
+  
+if (fs.existsSync(CONDA_INSTALL_DIR)) {
+  run('rm', `-rf \`cat ${CONDA_INSTALL_DIR}\``);
+  run('rm', `-f ${CONDA_INSTALL_DIR}`);
+}
```

Comment:
Maybe bash is better for this script?

Reply:
Do you mean to put the script in a bash file?

Reply:
I mean rewrite in Bash

Reply:
OK.